wled00/cfg.cpp

```diff
@@ -335,12 +335,16 @@ bool deserializeConfig(JsonObject doc, bool fromFS) {
   CJSON(irApplyToAllSelected, hw["ir"]["sel"]);
 
   JsonObject relay = hw[F("relay")];
+
+  if (relay.containsKey("odrain")) {
```

Comment:
Why not use common ArduinoJson construct `rlyOpenDrain  = relay[F("odrain")] | rlyOpenDrain;` or  `CJSON(rlyOpenDrain, relay[F("odrain")] )`?
It does the same.

Reply:
I just used the same syntax of existing "rev" setting. Should I change that too?

Reply:
That one has a negation included (for backwards compatibility).